src/components/dialog/edit-profile-dialog/EditProfileDialog.jsx

```diff
@@ -78,6 +89,24 @@ export const EditProfileDialog = ({
     }
   };
 
+  const handleFileInputChange = (event) => {
+    const selectedFile = event.target.files[0];
+
+    if (selectedFile) {
+      const reader = new FileReader();
+
+      reader.addEventListener("load", () => {
```

Comment:
check if there's an alternative for addEventListener here.

Reply:
yup! will look. And if I will find, will add in next PR.
Thanks for suggesting the change.